BUILD

```diff
@@ -3,3 +3,36 @@ load("@com_github_bazelbuild_buildtools//buildifier:def.bzl", "buildifier")
 buildifier(
     name = "buildifier",
 )
+
+# These are execution wrappers that buildfarm may choose to use when executing actions.
+# For their availability on a worker, they should be provided to a java_image as a "runtime_dep".
+# The relevant configuration for workers is the "execution policy".
+# That is where these binaries can be used and stacked.
+genrule(
+    name = "process-wrapper.binary",
+    srcs = ["@bazel//src/main/tools:process-wrapper"],
+    outs = ["process-wrapper"],
+    cmd = "cp $< $@;",
+)
+
+genrule(
+    name = "linux-sandbox.binary",
+    srcs = ["@bazel//src/main/tools:linux-sandbox"],
+    outs = ["linux-sandbox"],
+    cmd = "cp $< $@;",
```

Comment:
I did it this way because that's how the internal repository does it.  But you're right, we could move these to a different BUILD file also.  All that matters is that they are referenced correctly when we build the java_image (which we also do at the top level):  
```
java_image(
    name = ...,
    base = ...,
    classpath_resources = [...],
    entrypoint = [...],
    main_class = ...,
    runtime_deps = [
        ":as-nobody",
        ":process-wrapper.binary",
        ":tini.binary",
        "@build_buildfarm//src/main/java/build/buildfarm:shard-worker",
    ],
)
```

I assume these will need adjusted anyhow when we source them from upstream.